.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish Gem
+on:
+  release:
+    types:
+      - "created"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Publish to RubyGems
+        run: |
+          mkdir -p $HOME/.gem
+          touch $HOME/.gem/credentials
+          chmod 0600 $HOME/.gem/credentials
+          printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
```

Comment:
I'm presuming that these env vars, and when the `Publish Gem` workflow runs, is all configured separately in GitHub?

Reply:
Also, it looks like the PR still has the Travis build as a required check 😬

Reply:
Yes, The `GEM_HOST_API_KEY` is an env vars defined in the publish action that points to `secrets.RUBYGEMS_AUTH_TOKEN` which is been set in Github as a secret key.